internal/backend/remote-state/pg/backend_test.go

```diff
@@ -131,6 +141,8 @@ func TestBackendConfig(t *testing.T) {
 			Config: map[string]interface{}{
 				"conn_str":    connStr,
 				"schema_name": fmt.Sprintf("terraform_%s", t.Name()),
+				"table_name":  fmt.Sprintf("terraform_%s", t.Name()),
+				"index_name":  fmt.Sprintf("terraform_%s", t.Name()),
```

Comment:
Let's leave a few tests, where we don't set `table_name` and `index_name`

Reply:
Done! This required me to rework the way test gets config values.

I have added extra test cases:
- valid-config (nothing set)
- valid-config-with-table-name
- valid-config-with-index-name
- valid-config-with-table-name-and-index-name